Dockerfile

```diff
@@ -1,4 +1,5 @@
-ARG BASE_VERSION=3.2.0
+# syntax=docker/dockerfile:1
```

Comment:
I was able to do successful docker build without the use of this line (i.e. just deleted the line)